INSTALL.md

```diff
@@ -173,19 +230,36 @@ Debian/Ubuntu:
     $ node tileserver.js
     $ [Ctrl][A][D]
 
- You need a proxy that routes incoming requests.Configure the parallel running webserver (in most cases Apache) to listen on `127.0.0.1:8080`. A NodeJS proxy will redirect each request either to the webserver or the tileserver. Keep in mind that now 127.0.0.1 will appear as the user IP address - so you may have to change the configuration of geolocation or user statistics. Start the proxy in a screen session:
+## Enabling New Apache Configuration
 
- Remember to change the domains in the script and the configuration of your parallel running webservers. The NodeJS proxy listens on port 80 while parallel webservers should listen on 8080.
+ Now you enable the necessary Apache modules (mod_proxy, mod_proxy_http), enable the new virtual
+ hosts and restart Apache. On Ubuntu/Debian this is done using
 
-    $ cd ..
-    $ screen -R proxy
-    $ node proxy.js
-    $ [Ctrl][A][D]
+    $ sudo a2enmod proxy
+    $ sudo a2enmod proxy_http
+    $ cd /etc/apache2/site-available
+    $ sudo a2ensite www.openrailwaymap.org.conf
+    $ sudo a2ensite www.openrailwaymap.org.conf
+    $ sudo a2ensite tiles.openrailwaymap.org.conf
+
+ On other distros this is done by adding following entries to your Apache configuration (vhost do not have to be enabled because your Apache configuration contains `IncludeOptional /etc/apache2/vhosts.d/*.conf`. Check that the proxy modules are enabled. `httpd.conf should contain
+
+    LoadModule mod_proxy
+    LoadModule mod_proxy_http
+
+ Now reload (maybe restart?) Apache. On systems using systemd:
+
+    $ sudo systemctl reload apache2
```

Comment:
On CentOS 7 this is done with `sudo systemctl reload httpd`, on CentOS 6 with `sudo service httpd restart`. Could you add this?